collectors/build/sonar/src/main/java/com/capitalone/dashboard/collector/DefaultSonar6Client.java

```diff
@@ -36,7 +36,7 @@
 public class DefaultSonar6Client implements SonarClient {
     private static final Log LOG = LogFactory.getLog(DefaultSonar6Client.class);
 
-    private static final String URL_RESOURCES = "/api/components/search?qualifiers=TRK&ps=10000";
+    private static final String URL_RESOURCES = "/api/components/search?qualifiers=TRK&ps=500";
```

Comment:
500 should be PAGE_SIZE as you defined below?

Reply:
@tabladrum  yes, if page size is 10000, paging did not work. So I changed it to 500 as that is how Sonar defaults to max size.